### ISSUE TITLE
[EPIC] Migrate official compliance tests to Zuul

### ISSUE THREAD

**Issue by mbuechse** [SovereignCloudStack/standards#672]
PREREQUISITES:

- [x] #660
- [x] #715 
- [x] #716  
- [x] #664 
- [x] #665 
- [x] #738
- [x] #750 

OPTIONAL:

- [x] #671
- [x] #666 
- [ ] add notification mechanism to compliance monitor (or poll by some other service)

TASKS:

- [x] https://github.com/SovereignCloudStack/docs/issues/271
- [ ] mark GitHub actions as obsolete
- [x] make sure templates in compliance monitor are up to date
- [x] include generated table into docs page
- [x] include link to generated table into README.md
- [x] finally close https://github.com/SovereignCloudStack/docs/issues/166

**Comment by martinmo** [comment]
Motivated by https://github.com/SovereignCloudStack/standards/pull/680#issuecomment-2268320692, are parallel checks against the same cloud coordinated in Zuul, so they don't interfere with each other?

**Comment by berendt** [comment]
@martinmo We have worked with semaphores on jobs that basically share the same project and have overlapping names. This prevents jobs from running in parallel on a shared project.

Semaphore definition: https://github.com/regiocloud/e2e-tests/blob/main/.zuul.yaml#L82-L84
Semaphore use: https://github.com/regiocloud/e2e-tests/blob/main/.zuul.yaml#L107-L108

**Comment by martinmo** [comment]
@berendt Thanks for the hint! It seems we do not currently use this here, at least a quick `rg --hidden semaphore` through the standards repo didn't reveal anything like that.

**Comment by mbuechse** [comment]
I don't think we have the case right now (the nightly jobs are not run in parallel as far as I can tell), but it would be good to be on the safe side nevertheless.

**Comment by mbuechse** [comment]
Discussed in today's (extraordinary) meeting of SIG Std/Cert: most new features are actually not needed for the switch and can be postponed.
